static/src/javascripts/projects/commercial/modules/sticky-mpu.js

```diff
@@ -19,12 +19,15 @@ const onResize = (specs, _, iframe: ?HTMLElement) => {
     }
 };
 
-const stickyCommentsMpu = (adSlot: HTMLElement) => {
-    const dataName = adSlot.getAttribute('data-name');
-    if (dataName !== 'comments') {
-        return;
+const stickyMpuDataName = (adSlot: HTMLElement) => {
+    const dataName = adSlot.dataset.name;
+    if (dataName !== 'comments' || dataName !== 'right') {
```

Comment:
Actually I think this should be `&&` rather than `||`
as `!A && !B` is the negation of `A || B`.